documentation/python.py

```diff
@@ -907,47 +907,59 @@ def enclosing_module_for(state: State, name: str) -> Optional[str]:
     return None
 
 def add_module_dependency_for(state: State, object: Union[Any, str]):
+    """Update the set of module dependencies for `state.current_module`.
+
+    Based on the type of `object`, add the module it's defined in to the set of
+    dependencies for the current module. If the object is a string, it's assumed
+    to be a type name that's already in the name map and the module name is
+    extracted from it. If the object is a module, its name is extracted from
+    `object.__name__`. If the object is a class/function/enum/..., its module
+    name is extracted from `object.__module__`.
+    """
+
     assert state.current_module
 
     # If not a string and not a module, try looking if its name-mapped type is
     # in our name map. If it is, and its enclosing module is as well, use that
     # to have name mapping correctly applied for it.
-    name = None
+    module_name = None
```

Comment:
The variable rename feels unnecessary here, it drowns the other changes in noise.